src/components/DeviceItem/DeviceItem.tsx

```diff
@@ -1,61 +1,58 @@
 import { useMemo, useCallback } from "react";
-import { get, find } from "lodash";
-import { Title, TwoProperties } from "@deskpro/app-sdk";
+import { Title, Property, TwoProperties } from "@deskpro/app-sdk";
 import { useExternalLinks } from "../../hooks";
 import { getOS, getHumanCapacity } from "../../utils";
 import { ButtonAsLink, LansweeperLogo } from "../common";
 import type { FC } from "react";
-import type { Maybe } from "../../types";
-import type { Site, Device } from "../../services/lansweeper/types";
+import type { DeviceType } from "../../types";
 
 export type Props = {
-  device: Device;
-  sites: Site[];
-  siteId: Maybe<Site["id"]>;
+  device: DeviceType;
   onClickTitle?: () => void,
 };
 
-const DeviceItem: FC<Props> = ({ sites, siteId, device, onClickTitle }) => {
+const DeviceItem: FC<Props> = ({ device, onClickTitle }) => {
   const { getDeviceLink } = useExternalLinks();
-  const site = useMemo(() => find(sites, { id: siteId }), [sites, siteId]) as Maybe<Site>;
+  const site = device?.site;
```

Comment:
`device` is always an object based on typings, so no need for `?.`. Either change code or type.

Reply:
Yesterday in Slack you said the opposite

Reply:
You may have changed the typings. I was using the output from TS.